README.md

```diff
@@ -37,3 +37,187 @@ Following are some of the key features of *eProsima DDS Router*:
+* [Use Cases](https://eprosima-dds-router.readthedocs.io/en/latest/rst/use_cases/ros_cloud.html)
+* [Developer Manual](https://eprosima-dds-router.readthedocs.io/en/latest/rst/developer_manual/installation/sources/linux.html)
+* [Release Notes](https://eprosima-dds-router.readthedocs.io/en/latest/rst/notes/notes.html)
+
+
+## Installation Guide
+
+The instructions for installing the *DDS Router* application from sources and its required dependencies on a Linux
+environment are provided below. These installation instructions are a summarized version of the complete
+[installation guide](https://eprosima-dds-router.readthedocs.io/en/latest/rst/developer_manual/installation/sources/linux.html) available online, which also includes the [required steps](https://eprosima-dds-router.readthedocs.io/en/latest/rst/developer_manual/installation/sources/windows.html) for installing *DDS Router* on a Windows platform.
+
+### Requirements
+
+*eProsima DDS Router* requires the following tools to be installed in the system:
+* [CMake](https://cmake.org/), [g++](https://gcc.gnu.org/), [pip](https://pypi.org/project/pip/), [wget](https://www.gnu.org/software/wget/) and [git](https://git-scm.com/)
+* [Colcon](https://colcon.readthedocs.io/en/released/) [optional, not required for CMake-only installation]
+* [Gtest](https://github.com/google/googletest) [for test only]
+
+#### CMake, g++, pip, wget and git
+
+These packages provide the tools required to install DDS Router and its dependencies from command line. Install
+[CMake](https://cmake.org/), [g++](https://gcc.gnu.org/), [pip](https://pypi.org/project/pip/), [wget](https://www.gnu.org/software/wget/) and [git](https://git-scm.com/) using the package manager of the appropriate Linux distribution. For
+example, on Ubuntu use the command:
+
+```batch
+sudo apt install cmake g++ pip wget git
+```
+
+#### Colcon
+
+[colcon](https://colcon.readthedocs.io/en/released/) is a command line tool based on [CMake](https://cmake.org/) aimed at building sets of software packages. Install the ROS 2 development tools ([colcon](https://colcon.readthedocs.io/en/released/) and [vcstool](https://pypi.org/project/vcstool/)) by executing the following command:
+
+```batch
+pip3 install -U colcon-common-extensions vcstool
+```
+
+If this fails due to an Environment Error, add the `--user` flag to the `pip3` installation command.
+
+#### Gtest
+
+[Gtest](https://github.com/google/googletest) is a unit testing library for C++. By default, *DDS Router* does not
+compile tests. It is possible to activate them with the opportune [CMake options](https://colcon.readthedocs.io/en/released/reference/verb/build.html#cmake-options) when calling [colcon](https://colcon.readthedocs.io/en/released/) or
+[CMake](https://cmake.org/). For a detailed description of the Gtest installation process, please refer to the
+[Gtest Installation Guide](https://github.com/google/googletest).
+
+### Dependencies
+
+#### Asio and TinyXML2 libraries
```

Comment:
Add `yaml-cpp`

Reply:
Also added it to Linux and Windows documentation.